README.md

```diff
@@ -1,36 +1,36 @@
-# Gettext Translation Support for `mdbook`
+# Internationalization and Rendering extensions for `mdbook`
 
-[![Visit crates.io](https://img.shields.io/crates/v/mdbook-i18n-helpers?style=flat-square)](https://crates.io/crates/mdbook-i18n-helpers)
-[![Build workflow](https://img.shields.io/github/actions/workflow/status/google/mdbook-i18n-helpers/test.yml?style=flat-square)](https://github.com/google/mdbook-i18n-helpers/actions/workflows/test.yml?query=branch%3Amain)
-[![GitHub contributors](https://img.shields.io/github/contributors/google/mdbook-i18n-helpers?style=flat-square)](https://github.com/google/mdbook-i18n-helpers/graphs/contributors)
-[![GitHub stars](https://img.shields.io/github/stars/google/mdbook-i18n-helpers?style=flat-square)](https://github.com/google/mdbook-i18n-helpers/stargazers)
```

Comment:
Please keep the workflow status, the contributors and the stars (I want to keep highlighting celebrating those stats). I think it's okay that they're duplicated in the top-level and the nested README.